Scripts/Python/xJournalBookGUIPopup.py

```diff
@@ -212,3 +242,32 @@ def IsThereACover(self, bookHtml):
         if idx >= 0:
             return 1
         return 0
+
+    def ReadCurrentPage(self):
+        global BookJson
+        global CurrentPage
+        PtDebugPrint("xJournalBookGUIPopup ReadCurrentPage",level=kDebugDumpLevel)
+        BookJson = ptVault().findChronicleEntry(kBookChronicle)
+        if BookJson:
+            BookJson = json.loads(BookJson.getValue())
+        else:
+            BookJson = {}
```

Comment:
We probably shouldn't be hanging onto the value of the chronicle, in case some rogue vault manager comes along and changes the value of the chronicle while we're reading the journal. This would cause the vault manager's change to be discarded.